clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -213,6 +213,11 @@ Changes in existing checks
   when the format string is converted to a different type by an implicit
   constructor call.
 
+- Improved :doc:`modernize-use-std-format 
```

Comment:
Sorry, I missed existing `modernize-use-std-format` entry. Convention is to have single entry per check. Just merge existing statement with your text. At least previous release should have such examples.